code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -189,7 +208,7 @@
 	else //Just siphoning all air
 
 		var/transfer_moles = environment.total_moles()*(volume_rate/environment.volume)
-
+		transfer_moles = transfer_moles * transfer_modifier
```

Comment:
Ditto

Reply:
is the same for all the above, quick and easy way to do this. i'll do it properly later on. I left it in the PR mostly as a reminder